### PR TITLE
Add ability to exclude files that match a glob pattern

### PR DESCRIPTION
**Pull request by thetaiter**
This is my attempt at adding [this feature](https://github.com/jjjake/internetarchive/issues/544) myself. It appears to work, let me know if there is anything else I can do to make this change merge-worthy :)

### CODE REVIEW

**Comment by thetaiter** [comment]
I think I've gotten everything in order as far as the tests go at this point. Ready for review.

File: internetarchive/__version__.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = '3.0.2'
+__version__ = '3.0.3'
```
Comment (maxz) [inline internetarchive/__version__.py:1]: You should not change the version. That's what Jake does when preparing a new release.

**Review by maxz — APPROVED** [review]
Apart from the version bump, the pull request looks good.

File: internetarchive/__version__.py (hunk 1 above)
Comment (thetaiter) [reply internetarchive/__version__.py]: @maxz Sorry about that. This change has been reverted. Thanks!

**Comment by jjjake** [comment]
@thetaiter This is really great, and has been a long requested feature! Thanks so much for the contribution, we really appreciate it!

**Pull request merged**